spec/lib/kiev/request_body_filter_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "rack"
+
+describe Kiev::RequestBodyFilter do
+  describe ".filter" do
+    let(:request) do
+      Rack::Request.new(
+        Rack::MockRequest.env_for(
+          "/",
+          "REQUEST_METHOD" => "POST",
+          "CONTENT_TYPE" => "application/json",
+          :input => "{\"password\":12345}"
+        )
+      )
+    end
+
+    it "doesn't filter" do
```

Comment:
WDYT about adding a second context as well, when the filtering shouldn't happen?

Reply:
Added default case

Reply:
Aha, so we do have a default filter spec. Ok then